decidim-conferences/app/cells/decidim/conferences/participatory_spaces_cell.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    # This cell renders the participatory spaces card for an instance of a Participatory Space
```

Comment:
These docs seem wrong, as reading this it seems it receives a single participatory space as `model`, but it in fact receives a collection of participatory spaces. Can you add a line explaining this please? 😄